### DISCUSSION TITLE
how to remove front end login option

### DISCUSSION THREAD

**Discussion by mansfieled** [e107inc/e107#4920] [Q&A Core] [answered]
I'm new to e107, I like it, but finding it a little hard to navigate admin side. I'm used to seeing like global settings.
I want to build a site that does  not require a front end login  Just me as admin. So would like to remove that option

 any advice please /

 thanks very much

**Comment by Jimmi08** [comment, marked as answer]
In Settings/Preferences/ Basic Options/User Registration/Login

![image](https://user-images.githubusercontent.com/5429548/206872004-3b4279a0-0cb7-4468-b966-d3736143ea4d.png)

And maybe read this:
https://github.com/e107inc/e107/issues/4746#issuecomment-1265842770

**Reply by mansfieled** [reply]
Hi,

 thanks for the prompt reply

 I do have that set to disable, so  it removed the signup, but login is still there. I don't see that drop down your showing
have option to website members only, but that stops access to the site

 e107 v 2.3.2

admin theme bootstrap 3 v 1.0

**Reply by Jimmi08** [reply]
You need to restrict website to members only and then you see it.
![image](https://user-images.githubusercontent.com/5429548/206872896-09d21b2c-bd0d-4881-b726-fdd959192faf.png)

**Reply by Moc** [reply]
@mansfieled The steps provided by @Jimmi08 are the correct ones. Please be sure to use the latest version of e107, being v2.3.2. 

Are the steps working for you?

**Comment by mansfieled** [comment]
OK,but if I select that I cannot access the web site, one version says restricted site, the other says this page isn't working right now

**Comment by mansfieled** [comment]
Hi, Not sure if e107, maybe to complicated for what i need. 
Do you do some kind of paid support. like hour of training

**Comment by Jimmi08** [comment]
Hi, so again. I use 2.3.3 where that bug is already fixed.

e107 is perfect for member only site or only for backend site (for example, I use it as backend for some stuff on Wordpress site)

So you don't want member site but some case of backend only. 

1. set registration/login option off (so you will able to use admin login via e107_admin/admin.php) (login disabled)
2. you need to set member site only to hide frontend  parts like news, pages etc
3. there is bug in 2.3.2 with combination login disabled and splash page = login.php (you can fix this by selecting login only + login.php)
4. or you can select membersonly template and use admin path for login in. 

Another solution was described in that issue comment I linked.
